content/300-guides/500-other/880-troubleshooting-orm/050-creating-bug-reports.mdx

```diff
@@ -120,6 +120,18 @@ Please include any information about your environment and setup. Specifically it
 - Which **version of Prisma** you use (run `prisma -v` to see your Prisma version)
 - Which **version of Node.js** you use (run `node -v` to see your Node.js version)
 
+You can use the [`prisma debug`](/reference/api-reference/command-reference#debug) <span class="api"></span> command to retrieve debugging information.
+
+<Admonition>
+
+The `prisma debug` command available from version 5.6.0 and newer. If you're using an older version of Prisma, you can use this command by running:
```

Comment:
```suggestion
The `prisma debug` command is available from version `5.6.0` and newer. If you're using an older version of Prisma, you can use this command by running:
```

What's the usual formatting for versions? I thought we were using backticks, but it seems we don't always do that.

Reply:
I don't think we have strong opinions but we normally don't use backticks around for versions.